api/src/main/resources/metadata/Role_Privilege.xml

```diff
@@ -1401,6 +1404,57 @@
 
 
     <role_privilege  role="Organizational: Clinician"  privilege="View OrderTemplates"/>
+
+    <role_privilege  role="Cashier"  privilege="Adjust Cashier Bills"/>
+    <role_privilege  role="Cashier"  privilege="App: Access Cashier Tasks"/>
+    <role_privilege  role="Cashier"  privilege="App: stockmanagement.stockItems"/>
+    <role_privilege  role="Cashier"  privilege="App: View Cashier App"/>
+    <role_privilege  role="Cashier"  privilege="Manage Billing"/>
+    <role_privilege  role="Cashier"  privilege="Manage Cashier Bills"/>
+    <role_privilege  role="Cashier"  privilege="Manage Cashier Metadata"/>
+    <role_privilege  role="Cashier"  privilege="Manage Cashier Timesheets"/>
+    <role_privilege  role="Cashier"  privilege="Next Service Point"/>
+    <role_privilege  role="Cashier"  privilege="Purge Cashier Bills"/>
+    <role_privilege  role="Cashier"  privilege="Purge Cashier Metadata"/>
+    <role_privilege  role="Cashier"  privilege="Purge Cashier Timesheets"/>
+    <role_privilege  role="Cashier"  privilege="Task: Adjust Cashier Bills"/>
+    <role_privilege  role="Cashier"  privilege="Task: Cashier Timesheets"/>
+    <role_privilege  role="Cashier"  privilege="Task: Manage Cashier Metadata"/>
+    <role_privilege  role="Cashier"  privilege="Task: Manage Cashier Module"/>
+    <role_privilege  role="Cashier"  privilege="Task: View Cashier Reports"/>
+    <role_privilege  role="Cashier"  privilege="View Cashier Bills"/>
+    <role_privilege  role="Cashier"  privilege="View Cashier Metadata"/>
+    <role_privilege  role="Cashier"  privilege="View Cashier Timesheets"/>
+
+    <role_privilege  role="Organizational: Clinician"  privilege="Manage Patient Billing"/>
+    <role_privilege  role="Privilege Level: Full"  privilege="Manage Patient Billing"/>
+    <role_privilege  role="Privilege Level: High"  privilege="Manage Patient Billing"/>
+    <role_privilege  role="Triage"  privilege="Manage Patient Billing"/>
+
+    <role_privilege  role="Triage"  privilege="Adjust Cashier Bills"/>
+    <role_privilege  role="Triage"  privilege="App: View Cashier App"/>
```

Comment:
I think we could keep this in a separate role. Because we might not need it when Auto billing is implemented & also it might have security risks

Reply:
@akileng56 even this can be eliminated and we use Auto billing